packages/@okta/vuepress-site/okta-integration-network/index.md

```diff
@@ -22,7 +22,7 @@ guides:
     links:
       - text: What is SCIM?
         url: /docs/concepts/scim
-    docsUrl: /docs/guides/oin-build-provisioning-integration/overview/
+    docsUrl: /docs/guides/build-provisioning-integration/overview/
```

Comment:
Why is `oin` being removed here?

Reply:
The link is apparently actually supposed to be the one I changed it to and redirects right now, so I'm just updating it to match.

There is actually a separate problem for when it tries to smart-link it's broken 😅 You can see if you go to https://developer.okta.com/okta-integration-network/ and try to click the "View Docs" link under "Automate Provisioning". If you regular click, it will show 404, but if you Shift/Cmd Click to open in a new tab/window, it will work and redirect.

Reply:
FYI @jakubvul on this link fix